test/validators.js

```diff
@@ -9524,18 +9524,30 @@ describe('Validators', () => {
       {
         locale: 'dv-MV',
         valid: [
-          '+960973256874',
-          '781246378',
-          '+960766354789',
-          '+960912354789',
+          '+9609112345',
+          '+9609958973',
+          '+9607258963',
+          '+9607958463',
+          '9609112345',
+          '9609958973',
+          '9607212963',
+          '9607986963',
```

Comment:
Are these really valid? It seems strange to have a country code there without any indicator that it is a country code (`+` or `00`, for example). I couldn't find any examples of numbers like this from your links.

Reply:
Thanks for your comment, as you make a valid point.

Strictly speaking:
no, these are **not** valid phone numbers, if there is no `+` or `00` in front of the country code.

However, I just tried to stick to the same RegExp style like (most if not almost all) the other phone numbers, which are making the `+` optional, when you don't set the `strictMode` option to `true`.
And I just included these _technically invalid_ phone numbers in the tests as well, to test that this "optional" matching works as well :-)

E.g. random examples from [isMobilePhone.js:](https://github.com/validatorjs/validator.js/blob/master/src/lib/isMobilePhone.js) 
* `nn-NO` currently uses `/^(\+?47)?[49]\d{7}$/`
* `el-GR` currently used `/^(\+?30|0)?(69\d{8})$/`

both of these will match phone numbers without `00` or `+` as valid.

a big side note on this topic
---

Generally speaking the isMobilePhone RegExp do not seem to be very constistent altogether in the way the RegExp are matching the phone numbers:

* some are checking for the + as mandatory (regardless of strictMode)
* some are checking for the + as optional
* many do not check for the 00 prefix, which actually should be valid as well
* etc.
* 
There's a whole range of different formats in the isMobilePhone, which IMHO should be made a bit more consistent, across all of the different countries (or locales in this case).

However fixing that would be a whole new thingon its own, which I could try to tackle soon as well - the only problem I see is that there are a lot of unmerged PRs in regards to the isMobilePhone, so I am not sure if I should already try to add my changes or wait until these are added?

Reply:
That is certainly interesting. I don't use phone validation from this library, so I haven't looked much into that side of things. This should definitely be fixed somehow. Could you post this as a new issue?

Reply:
Sure, I'll write something up later today and post as a separate issue, so that we can track this better

Reply:
Sure, let's track this as a separate issue, did you post it? Will definitely be a breaking change but a necessary one.

Reply:
@profnandaa yeah, had posted it as #2124